recipes/nbpresent/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.0.1" %}
+{% set gh_org = "Anaconda-Platform" %}
```

Comment:
It's OK to template whatever you want,  but doing the organization and repo here seems a little overboard. I hope those aren't going to change often!